docker/four-nodes.yml

```diff
@@ -2,10 +2,11 @@ version: '3'
 services:
   core0:
     environment:
+      RADIXDLT_HOST_IP_ADDRESS: core0
       RADIXDLT_NETWORK_SEEDS_REMOTE: core1,core2,core3
       RADIXDLT_CONSENSUS_FIXED_NODE_COUNT: 4
       #RADIXDLT_NETWORK_DISCOVERY_URLS: https://sunstone.radixdlt.com:8443/explorer/node-finder
-      JAVA_OPTS: -server -Xms1536m -Xmx1536m -XX:+HeapDumpOnOutOfMemoryError -XX:+AlwaysPreTouch -Djava.security.egd=file:/dev/urandom -Dcom.sun.management.jmxremote.port=9011 -Dcom.sun.management.jmxremote.rmi.port=9011 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=localhost -agentlib:jdwp=transport=dt_socket,address=50505,suspend=n,server=y
+      JAVA_OPTS: -server -Xms1g -Xmx1g -XX:+HeapDumpOnOutOfMemoryError -XX:+AlwaysPreTouch -Djava.security.egd=file:/dev/urandom -Dcom.sun.management.jmxremote.port=9011 -Dcom.sun.management.jmxremote.rmi.port=9011 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=localhost -agentlib:jdwp=transport=dt_socket,address=50505,suspend=n,server=y
```

Comment:
Reduced to 1Gib for all single, two, three and four node files.  Based on memory profiling, this should be more than sufficient, and will allow us to run more hosts if needed.